Source/Scene/Camera.js

```diff
@@ -2166,6 +2166,69 @@ define([
     };
 
     /**
+     * Sets the camera so that the current view completely contains the provided bounding sphere.
+     *
+     * @param {BoundingSphere} boundingSphere The bounding sphere to view.
+     * @param {Object} [options] Object with the following properties:
+     */
+    Camera.prototype.viewBoundingSphere = function(boundingSphere, options) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(boundingSphere)) {
+            throw new DeveloperError('boundingSphere is required.');
+        }
+        //>>includeEnd('debug');
+
+        var transform = Transforms.eastNorthUpToFixedFrame(boundingSphere.center);
+        this.transform = transform;
+        var r = 2.0 * Math.max(boundingSphere.radius, this.frustum.near);
```

Comment:
Did we get this from the code I wrote for the Model Sandcastle example?  That was a hack.  I believe we need to take into account the field of view (at least for 3D/CV)...and test across all scene modes.  It is still pretty trivial.